### PR TITLE
Add a hook for publicsuffix2

### PR DESCRIPTION
**Pull request by mhils**
And here's the next one! 😃

I will send PRs sequentially, as there's otherwise a merge conflict on each PR.

### CODE REVIEW

**Comment by bwoodsend** [comment]
> I will send PRs sequentially, as there's otherwise a merge conflict on each PR.

Yes, please do...

File: src/_pyinstaller_hooks_contrib/tests/test_libraries.py [hunk 1]
```diff
@@ -317,3 +317,10 @@ def test_passlib(pyi_builder):
     pyi_builder.test_source("""
         import passlib.apache
         """)
+
+
+@importorskip('publicsuffix2')
+def test_publicsuffix2(pyi_builder):
+    pyi_builder.test_source("""
+        import publicsuffix2
```
Comment (bwoodsend) [inline src/_pyinstaller_hooks_contrib/tests/test_libraries.py:325]: This doesn't appear to be enough. I can compile and run this without the hook and it still works. Try:
```suggestion
        import publicsuffix2
        publicsuffix2.PublicSuffixList()
```
That gives me the error without the hook.

**Review by bwoodsend — CHANGES_REQUESTED** [review]

File: src/_pyinstaller_hooks_contrib/tests/test_libraries.py (hunk 1 above)
Comment (mhils) [reply src/_pyinstaller_hooks_contrib/tests/test_libraries.py:325]: Good point, thanks. How do you go on to test this? I naively expected the pyi_builder thing to throw an error if execution succeeds in absence of a hook.

**Review by bwoodsend — APPROVED** [review]
Perfect

**Comment by bwoodsend** [comment]
As usual, @Legorooj gets the final say on this...

**Review by Legorooj — APPROVED** [review]
Nice work and thanks!

**Pull request merged**